src/main/kotlin/no/nav/k9/los/tjenester/saksbehandler/oppgave/OppgaveApisTjeneste.kt

```diff
@@ -90,38 +92,56 @@ class OppgaveApisTjeneste(
         }
     }
 
-    suspend fun endreReservasjon(
+    suspend fun endreReservasjoner(
         reservasjonEndringDto: ReservasjonEndringDto,
         innloggetBruker: Saksbehandler
+    ) {
+        reservasjonEndringDto.oppgaveNøkkel.forEach { oppgaveNøkkel ->
+            endreReservasjon(
+                innloggetBruker,
+                oppgaveNøkkel,
+                reservasjonEndringDto.brukerIdent,
+                reservasjonEndringDto.reserverTil,
+                reservasjonEndringDto.begrunnelse
+            )
+        }
+    }
+
+    private suspend fun endreReservasjon(
+        innloggetBruker: Saksbehandler,
+        oppgaveNøkkel: OppgaveNøkkelDto,
+        brukerIdent: String? = null,
```

Comment:
Synes denne fortjener et bedre navn. Hvem sin brukerident er dette? Navnet burde tydeliggjøre om det er innlogget bruker eller den som skal motta reservasjonen

Reply:
Ser at innlogget bruker også er et parameter her, så det er ikke like kryptisk, men synes poenget står seg

Reply:
Jeg endret til `tilBrukerIdent`, så at litt lenger ned i koden er det brukt `tilSaksbehandler` for en variabel som er hentet fra repository. Andre steder ser jeg imidlertid det er brukt `ny...` som prefiks, for eksempel `nySaksbehandlerId`.